src/store/user/user-context.js

```diff
@@ -10,17 +10,30 @@ export const UserContextProvider = ({ children }) => {
   const [showMemberRoleUpdateModal, setShowMemberRoleUpdateModal] =
     useState(false);
   const [selectedMember, setSelectedMember] = useState(null);
+  const [showMemberTaskUpdateModal, setShowMemberTaskUpdateModal] =
+    useState(false);
+  const [isNoteworthy, setIsNoteworthy] = useState(false);
+  const [isOptionKey, setIsOptionKey] = useState(false);
+  const [taskId, setTaskId] = useState('');
```

Comment:
Should this belong in the `user-context`?

Reply:
We were having the showModal context here only so I thought that it could be there but you are right we can make a different context for it